pkg/commands/git.go

```diff
@@ -471,11 +473,12 @@ func (c *GitCommand) RebaseMode() (string, error) {
 func (c *GitCommand) DiscardAllFileChanges(file *File) error {
 	// if the file isn't tracked, we assume you want to delete it
 	quotedFileName := c.OSCommand.Quote(file.Name)
-	if file.HasStagedChanges {
+	if file.HasStagedChanges || file.HasMergeConflicts || file.HasInlineMergeConflicts {
```

Comment:
sorry to do this to you but I just realised hasInlineMergeConflicts is always true with hasMergeConflicts is true, so we can remove that here
```suggestion
	if file.HasStagedChanges || file.HasMergeConflicts {
```

Reply:
good point, addressing it

Reply:
pushed a new commit removing it from the if condition